app/controllers/voice/otp_controller.rb

```diff
@@ -27,7 +27,8 @@ def code
     end
 
     def message
-      t('voice.otp.message', code: code_with_pauses)
+      expiration = Devise.direct_otp_valid_for.to_i / 60
```

Comment:
I'm sure I'm just missing something, but where is direct_otp_valid_for defined?

Reply:
It's a Devise config which is set in the initializer here: https://github.com/18F/identity-idp/blob/master/config/initializers/devise.rb#L28

Reply:
👍